GitUI/CommandsDialogs/FormDiff.cs

```diff
@@ -137,9 +137,10 @@ private void openWithDifftoolToolStripMenuItem_Click(object sender, EventArgs e)
                 diffKind = GitUI.RevisionDiffKind.DiffAB;
             }
 
-            foreach (var selectedItem in DiffFiles.SelectedItems)
+            foreach (var itemWithParent in DiffFiles.SelectedItemsWithParent)
             {
-                RevisionGrid.OpenWithDifftool(selectedItem.Name, selectedItem.OldName, diffKind, selectedItem.IsTracked);
+                IList<GitRevision> revs = new List<GitRevision> { DiffFiles.Revision, itemWithParent.ParentRevision };
+                RevisionGrid.OpenWithDifftool(revs, itemWithParent.Item.Name, itemWithParent.Item.OldName, diffKind, itemWithParent.Item.IsTracked);
```

Comment:
Is it worth checking `itemWithParent.Item != null`?
From the context it isn't obvious